spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/embedded/tomcat/TldSkipPatterns.java

```diff
@@ -50,6 +50,7 @@
 		patterns.add("h2*.jar");
 		patterns.add("hamcrest*.jar");
 		patterns.add("hibernate*.jar");
+		patterns.add("jaxb-*.jar");
```

Comment:
Can we please consider narrowing that down to `jaxb-runtime-*.jar`?

Reply:
I think perhaps this can be tightened to `jaxb-runtime*.jar` but not `jaxb-runtime-*.jar`?

Reply:
Either one seems to work but I don't understand why jaxb-runtime-*.jar works because the string "jaxb-runtime-" does not seem to appear anywhere that I can see?